src/components/Home/Home.jsx

```diff
@@ -27,13 +29,59 @@ const Home = () => {
 	// console.log('User:', user);
 
 	return (
-		<div>
+		<>
 			{user ? (
-				<h2>
-					Witaj {user.firstName} {user.lastName}!
-				</h2>
+				<div>
+					<div>
+						<h2>
+							Witaj {user.firstName} {user.lastName}!
+						</h2>
+
+						<div className={styles.honeycomb}>
+							<div className={styles.ibwsfix}>
+								<div className={styles.hexagon}>
+									<div className={styles.hexagontent}>
+										<Link to='/userprofile'>Mój profil</Link>
+									</div>
+								</div>
+								<div className={styles.hexagon}>
+									<div className={styles.hexagontent}>
+										<Link to='/mytrips'>Moje podróże</Link>
+									</div>
+								</div>
+								<div className={styles.hexagon}>
+									<div className={styles.hexagontent}>
+										<Link to='/addtrip'>Stwórz podróż</Link>
+									</div>
+								</div>
+								<div className={styles.hexagon}>
+									<div className={styles.hexagontent}>
+										<Link to='/alltrips'>Szukaj podróży</Link>
+									</div>
+								</div>
+								<div className={styles.ibwsfix}>
```

Comment:
Jak usunę tutaj tę klasę, to też działa, ale póki co nic bym z tym nie robiła, bo zobaczymy jeszcze co wymyśli team UX.

Reply:
ta klasa ma jakiś wpływ na późniejsze układanie sie plastrów - jak nie rozminię tego - to usunę bo sama też miałam wątpliwości w tym miejscu